packages/vue_frontend_rpc/src/assets/css/globalStyles.css

```diff
@@ -168,9 +168,9 @@ form.inquirer-gui i.theme--light.v-icon {
     animation: none;
 }
 
-.v-textarea > .v-input__control {
+/* .v-textarea > .v-input__control {
```

Comment:
redundant?

Reply:
seems like. but I prefer to leave it meanwhile as a reference until final ux confirmation